src/quo/components/share/share_qr_code/view.cljs

```diff
@@ -153,7 +160,7 @@
          {:color           colors/white-opa-40
           :container-style style/watched-account-icon}])]
      [share-button {:on-press on-share-press}]]
-    (when (not= share-qr-type :profile)
+    (when (not (contains? #{:profile :channel} share-qr-type))
```

Comment:
If it's easy to determine, have a positive if condition, ie. `(when (= % :account))`.

If not easy to determine, a more explicit `or` is preferable. `contains?` feels fragile.

Reply:
Update: It's not easy to determine an exhaustive list. A negative filter is fine.

![image](https://github.com/status-im/status-mobile/assets/1925158/ec3363ff-0e6e-4a0a-92b6-bd6959155f87)